test/sandbox/sandbox.js

```diff
@@ -1,3 +1,35 @@
+if (!Object.assign) {
```

Comment:
`Object.assign` is only used in the compiler, but we may want/need to include a shim for it. We could use the babel ES5/6 polyfill, but that might be a bit heavy.